### PR TITLE
project-chatbot-semla

### PR DESCRIPTION
**Pull request by idsab**
## Netlify link

https://app.netlify.com/sites/thesemelbot/overview

## Collaborators

[Lovisaaberg, pernillasterner]

### CODE REVIEW

File: .DS_Store
Comment (NellaMori) [inline .DS_Store:1]: .DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file (if you don't have one, you need to create it) 👍

File: code/assets/.DS_Store
Comment (NellaMori) [inline code/assets/.DS_Store:1]: same here 🧹

File: code/.DS_Store
Comment (NellaMori) [inline code/.DS_Store:1]: same here 🧹

File: code/script.js [hunk 1]
```diff
@@ -1,53 +1,231 @@
...
+  yesButton.addEventListener("click", () => {
+    showMessage("Confirm order", "user");
+    setTimeout(() => {
+      yesButton.style.display = "none";
+      noButton.style.display = "none";
+      showMessage(
+        "We will prepare your order ☕️ Thank you for using the Semel Bot!",
+        "bot"
+      );
+      sound.play();
```
Comment (NellaMori) [inline code/script.js:212]: such a nice touch 🎶

**Review by AntonellaMorittu — APPROVED** [review]
Hello Ida, Lovisa & Pernilla, wow love your chatbot! Works very well and meets the requirements, the code is clean and the flow is _flowless_ 😄 It shows you had fun and put into practice all the skills learnt so far, kudos! 🥳 I left a few comments to clean up some files 🧹 👍

**Pull request closed without merging**